presentation/src/main/java/com/pocs/presentation/mapper/PostDetailMapper.kt

```diff
@@ -9,6 +9,7 @@ fun PostDetail.toUiState() = PostDetailItemUiState(
     title = title,
     content = content,
     views = views,
+    onlyMember = true,
```

Comment:
여기는 아마 오타인것같네요...? 기본값 true는 생성할때 만 부여하기로 했네요.
```suggestion
    onlyMember = onlyMember,
```